examples/custom_backbones/squeezenet/cifar10.yml

```diff
@@ -0,0 +1,89 @@
+experiment_name: squeezenet_CIFAR10
+logging: None
+dataset: {
+  train: {
+    dataset: CIFAR10,
+    args: {
+      root: datasets,
+      train: True,
+      download: True
+    },
+  },
+  eval: {
+    dataset: CIFAR10,
+    args: {
+      root: datasets,
+      train: False,
+      download: True
+    }
+  },
+  dataloader: {
+    dataloader: DataLoader,
+    args: {
+      num_workers: 0,
+      batch_size: 256,
+      shuffle: True,
+    },
+  },
+}
+model: {
+  name: softmax,
+  network_args: {
+    backbone: squeezenetv1.1,
+    pretrained: True,
+    num_classes: 10,
+  },
+  preprocess_args: {
+    input_size: 224,
+    input_normalization: {
+      mean: [0.4914, 0.4822, 0.4465],
+      std: [0.2023, 0.1994, 0.2010],
+      scaler: 255,
+    }
+  },
+  loss_args: {
+    reduction: mean
+  },
+  postprocess_args: {}
+}
+trainer: {
+  optimizer: {
+    method: SGD,
+    args: {
+      lr: 0.0194,
+      momentum: 0.9,
+      weight_decay: 0.0005,
+    }
+  },
+  scheduler : {
```

Comment:
I think it's better not to use scheduler in this example because of small epoch number, in my case this can make gradient explosion (nan loss). I think using lr=0.001 without any scheduler work just fine

Reply:
fixed